src/main/msp/msp.c

```diff
@@ -2698,7 +2698,7 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
 #endif
     case MSP_SET_ARMING_CONFIG:
         armingConfigMutable()->auto_disarm_delay = sbufReadU8(src);
-        sbufReadU8(src); // reserved
+        armingConfigMutable()->gyro_cal_on_first_arm = sbufReadU8(src); // reserved
```

Comment:
I suppose now the comment is not necessary. Someone knows what was the reserved for?

Reply:
It seems it was for the `disarm_kill_switch` https://github.com/betaflight/betaflight/commit/45a6588eaf36788ac8acfa81dae4fbad11cf39be

It's a good idea to reuse old MSP fields?

Reply:
@McGiverGim the parameter was removed in firmware 3.x as configurator only supports firmware 4.x so we can reuse safely.